evm/v0.5/contracts/dev/Owned.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity 0.5.0;
+
+/**
+ * @title The Owned contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract Owned {
+
+  address public owner;
+  address private pendingOwner;
+
+  event OwnershipTransferRequested(
+    address indexed to,
+    address from
+  );
+  event OwnershipTransfered(
+    address indexed to,
+    address from
+  );
+  event OwnershipRenounced(
+    address indexed by
+  );
+
+  constructor() public {
+    owner = msg.sender;
+  }
+
+  /**
+   * @dev Allows an owner to begin transferring ownership to a new address,
+   * pending.
+   */
+  function transferOwnership(address _to)
+    public
+    onlyOwner()
+  {
+    pendingOwner = _to;
+
+    emit OwnershipTransferRequested(_to, owner);
+  }
+
+  /**
+   * @dev Allows an ownership transfer to be completed by the recipient.
+   */
+  function acceptOwnership()
+    public
+  {
+    require(msg.sender == pendingOwner, "Must be requested to accept ownership");
+
+    address oldOwner = owner;
+    owner = msg.sender;
+    pendingOwner = address(0);
+
+    emit OwnershipTransfered(msg.sender, oldOwner);
+  }
+
+  /**
+   * @dev Renounces ownership so that no one owns the contract.
+   */
+  function renounceOwnership()
```

Comment:
I'm actually against having a `renounceOwnership` function for our needs. Why would any of our contracts need to operator outside of ownership? I think it would open up new problems like locking funds in contracts, especially if this function was accidentally called by the owner.

Reply:
Went this route. We could always make a simple contract to accept ownership if we wanted to burn it.